sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -69,8 +70,127 @@ public enum ServiceVersion
+                "x-ms-blob-append-offset",
+                "x-ms-blob-cache-control",
+                "x-ms-blob-committed-block-count",
+                "x-ms-blob-condition-appendpos",
+                "x-ms-blob-condition-maxsize",
+                "x-ms-blob-content-disposition",
+                "x-ms-blob-content-encoding",
+                "x-ms-blob-content-language",
+                "x-ms-blob-content-length",
+                "x-ms-blob-content-md5",
+                "x-ms-blob-content-type",
+                "x-ms-blob-public-access",
+                "x-ms-blob-sequence-number",
+                "x-ms-blob-type",
+                "x-ms-copy-destination-snapshot",
+                "x-ms-creation-time",
+                "x-ms-default-encryption-scope",
+                "x-ms-delete-snapshots",
+                "x-ms-delete-type-permanent",
+                "x-ms-deny-encryption-scope-override",
+                "x-ms-encryption-algorithm",
+                "x-ms-if-sequence-number-eq",
+                "x-ms-if-sequence-number-le",
+                "x-ms-if-sequence-number-lt",
+                "x-ms-incremental-copy",
+                "x-ms-lease-action",
+                "x-ms-lease-break-period",
+                "x-ms-lease-duration",
+                "x-ms-lease-id",
+                "x-ms-lease-time",
+                "x-ms-page-write",
+                "x-ms-proposed-lease-id",
+                "x-ms-range-get-content-md5",
+                "x-ms-rehydrate-priority",
+                "x-ms-sequence-number-action",
+                "x-ms-sku-name",
+                "x-ms-source-content-md5",
+                "x-ms-source-if-match",
+                "x-ms-source-if-modified-since",
+                "x-ms-source-if-none-match",
+                "x-ms-source-if-unmodified-since",
+                "x-ms-tag-count",
+                "x-ms-encryption-key-sha256"
+            };
+
+            List<string> LoggedQueryParameters = new List<string>
+            {
+                "comp",
+                "maxresults",
+                "rscc",
+                "rscd",
+                "rsce",
+                "rscl",
+                "rsct",
+                "se",
+                "si",
+                "sip",
+                "sp",
+                "spr",
+                "sr",
+                "srt",
+                "ss",
+                "st",
+                "sv",
+                "include",
+                "marker",
+                "prefix",
+                "copyid",
+                "restype",
+                "blockid",
+                "blocklisttype",
+                "delimiter",
+                "prevsnapshot",
+                "ske",
+                "skoid",
+                "sks",
+                "skt",
+                "sktid",
+                "skv",
+                "snapshot"
+            };
+
+            LoggedHeaderNames.ForEach(header => Diagnostics.LoggedHeaderNames.Add(header));
```

Comment:
Can we call `Diagnostics.LoggedHeaderNames.Add(...)` on each value directly instead of allocating the list and a closure?  A [multi-cursor selection in VS or Code](https://blogs.msdn.microsoft.com/mschray/2014/09/03/multi-cursor-in-visual-studio/) would make short work of this.